### PR TITLE
chore: Updated change log and readme.

### PR DESCRIPTION
**Pull request by rashidsp**
## Summary
- Updated Readme and change log for Event Processor and Optimizely close.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
-## [Unreleased]
-Changes that have landed but are not yet released.
+## 3.3.0
+September 3rd, 2019
 
 ### New Features:
 - Added non-typed `get_feature_variable` method ([#190](https://github.com/optimizely/ruby-sdk/pull/190)) as a more idiomatic approach to getting values of feature variables.
   - Typed `get_feature_variable` methods will still be available for use.
+- Added support for event batching via the event processor.
+- Events generated by methods like `activate`, `track`, and `is_feature_enabled` will be held in a queue until the configured batch size is reached, or the configured flush interval has elapsed. Then, they will be combined into a request and sent to the event dispatcher.
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: nit. ~~combined into a request~~ batched into a single payload

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
 # Optimizely Ruby SDK Changelog
 
-## [Unreleased]
-Changes that have landed but are not yet released.
+## 3.3.0
+September 3rd, 2019
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: nit. Change this to 4th (or may be 5th) now.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
 
 ### New Features:
 - Added non-typed `get_feature_variable` method ([#190](https://github.com/optimizely/ruby-sdk/pull/190)) as a more idiomatic approach to getting values of feature variables.
   - Typed `get_feature_variable` methods will still be available for use.
+- Added support for event batching via the event processor.
+- Events generated by methods like `activate`, `track`, and `is_feature_enabled` will be held in a queue until the configured batch size is reached, or the configured flush interval has elapsed. Then, they will be combined into a request and sent to the event dispatcher.
+- To configure event batching, set the `batch_size` and `flush_interval` properties in the `OptimizelyFactory` using `OptimizelyFactory.max_event_batch_size(batch_size, logger)` and `OptimizelyFactory.max_event_flush_interval(flush_interval, logger)` and then create `OptimizelyFactory.custom_instance`.
+- Event batching is enabled by default. `batch_size` defaults to `10`. `flush_interval` defaults to `30000` Milliseconds.
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: nit. milliseconds (don't capitalize the m)

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
+- Added the `close` method representing the process of closing the instance. When `close` is called, any events waiting to be sent as part of a batched event request will be immediately batched and sent to the event dispatcher.
+### Deprecated
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: nit. Insert new line above this.

File: README.md [hunk 5]
```diff
@@ -128,6 +129,48 @@ The following properties can be set to override the default configurations for `
...
+)
+~~~~~~   
+
+#### Advanced configuration
+The following properties can be used to customize the `BatchEventProcessor` configuration.
+
+| **Property Name** | **Default Value** | **Description**
+| -- | -- | --
+| `event_queue` | 1000 | `SizedQueue.new(100)` or `Queue.new`. Queues individual events to be batched and dispatched by the executor. Default value is 1000.
+| `event_dispatcher` | nil | Used to dispatch event payload to Optimizely.
```
Comment (aliabbasrizvi) [inline README.md]: Let's comment on what this is set to if it is nil.

File: CHANGELOG.md (hunk 4 above)
Comment (rashidsp) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,11 +1,20 @@
 # Optimizely Ruby SDK Changelog
 
-## [Unreleased]
-Changes that have landed but are not yet released.
+## 3.3.0
+September 5th, 2019
```
Comment (aliabbasrizvi) [inline CHANGELOG.md]: Can you update this date and we can merge this and release

**Pull request merged**